src/tests/Interop/ICastable/Castable.csproj

```diff
@@ -8,4 +8,10 @@
   <ItemGroup>
     <Compile Include="Castable.cs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)/src/tests/Common/XUnitWrapperGenerator/XUnitWrapperGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
Could you please integrate the tiny delta I was experimenting with on top of your change in https://github.com/dotnet/runtime/pull/61224 so that we don't need to explicitly reference the Roslyn generator in each test being converted?